src/App.tsx

```diff
@@ -1,10 +1,56 @@
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useEffect, useMemo, useState } from 'react';
+import cn from 'classnames';
 import { UserWarning } from './UserWarning';
-
-const USER_ID = 0;
+import { Todo } from './types/Todo';
+import { getTodos } from './api/todos';
+import { FilterType } from './types/FilterType';
+import { Error } from './types/Error';
+import { TodoList } from './components/TodoList';
+import { TodoFilter } from './components/TodoFilter';
+import { TodoError } from './components/TodoError';
+
+const USER_ID = 11260;
+
+const getVisibleTodos = (todos: Todo[], status: FilterType) => {
+  return todos
+    .filter(todo => {
+      switch (status) {
+        case FilterType.Completed:
+          return todo.completed;
+
+        case FilterType.Active:
+          return !todo.completed;
+
+        default:
+          return true;
+      }
+    });
+};
```

Comment:
Not mistake in this task, but for future:
No need to filter in case `FilterType` is `All`